azure_devops_rust_api/examples/artifact_provenance.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// artifact_provenance.rs
+// Artifacts list example.
```

Comment:
Please could you update this comment to say: "Artifacts provenance example" and add a little more detail to explain what values you get back when querying provenance.

Reply:
Looks good - thanks!